src/cards/top-languages-card.js

```diff
@@ -276,6 +283,7 @@ const renderTopLanguages = (topLangs, options = {}) => {
     langs_count = DEFAULT_LANGS_COUNT,
     border_radius,
     border_color,
+    disable_animations,
```

Comment:
Hey @Zo-Bro-23 can you pull the changes from master? Why is there some different changes in diff like these?

Reply:
@Zo-Bro-23 I think you need to run `npm run format` and rebase on the current master branch. I had to do the same for my PR.

Reply:
@Zo-Bro-23 nevermind I already fixed it. 👍🏻

Reply:
It was @rickstaa's commits, and I don't understand how he can push to my fork, but it should be fixed now. Thanks @rickstaa!

Reply:
> It was @rickstaa's commits, and I don't understand how he can push to my fork, but it should be fixed now. Thanks @rickstaa!

Yea, I don't know what went wrong with my comment, but I fixed it 😅.